### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by zyplos** [Zyplos/proof-tool#3]
There are certain accessibility features that should be added for user's benefit.

- [ ] A way to change the base font size
- [ ] High contrast mode
- [ ] This entire webapp should be useable with a keyboard alone.
